lib/common/zstd_internal.h

```diff
@@ -196,6 +196,13 @@ static void ZSTD_copy8(void* dst, const void* src) { memcpy(dst, src, 8); }
 #define COPY8(d,s) { ZSTD_copy8(d,s); d+=8; s+=8; }
 static void ZSTD_copy16(void* dst, const void* src) { memcpy(dst, src, 16); }
 #define COPY16(d,s) { ZSTD_copy16(d,s); d+=16; s+=16; }
+static void ZSTD_copy32(void* dst, const void* src) { memcpy(dst, src, 32); }
+
+#if defined(__clang__)
```

Comment:
Workaround for Clang since Clang miscompiles '{ ZSTD_copy32(d,s); d+=32; s+=32; }', produces binary which fails to decompress files correctly.